### PR TITLE
Bump version to v0.4.0

### PR DESCRIPTION
**Pull request by rhattersley**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+{% set version = "0.4.0" %}
 
 package:
   name: pyepsg
   version: {{ version }}
 
 source:
   url: https://github.com/rhattersley/pyepsg/archive/v{{ version }}.tar.gz
-  sha256: 2af92df0f398db458485b76d58ba8303202c93ed30ccd27c4c6463e90b4a79d2
+  sha256: 2737302c3661464a867d1844ab1adab046ccbf763edd54140811e21238894f05
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @rhattersley if you upload a source tarball to PyPI we can use that and avoid this checksum changes from GitHub.

Comment (rhattersley) [reply recipe/meta.yaml]: The source distribution is now on PyPI (it was supposed to have been all along, but twine fell over after the wheel). But, sorry - I don't understand how using PyPI avoids checksum changes. 😕 I found [this](https://conda-forge.org/docs/meta.html#populating-the-hash-field) in the docs but it didn't really help.

Comment (ocefpaf) [reply recipe/meta.yaml]: GitHub changes checksum if you make changes in the branch where the releases was based on or if you edit the release notes. PyPI will has the tarball and that is it. (We can upload a tarball to GitHub releases download too but that usually is an extra step that no one does.)

**Comment by rhattersley** [comment]
Thanks @ocefpaf 👍

**Comment by ocefpaf** [comment]
Thanks for the new release!

**Pull request merged**